doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,7 +24,31 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-*
+* LwM2M:
```

Comment:
Can you move this whole section under a "Networking" header?

Reply:
Yes, done